docs/src/reference-guide/components/graphql/queries.md

```diff
@@ -0,0 +1,166 @@
+# Queries
```

Comment:
@deekerno 
- I think we can remove this. 
- Many (if not all) users expect this to be default behavior. 
- If it turns out we need to explain this we can revive this, but at present, it looks like another piece of documentation that demonstrates functionality most users will expect by default.